docs/languages/css.md

```diff
@@ -104,6 +104,17 @@ You can extend VS Code's CSS support through a declarative [custom data format](
 
 You can read more about using custom data in the [vscode-custom-data](https://github.com/microsoft/vscode-custom-data) repository.
 
+## Formatting
+
+The CSS Languages Features extension also providesa formatter. The formatter works with CSS, LESS and SCSS. It is implemented by the [JS Beautify library](https://github.com/beautify-web/js-beautify) and comes with the following settings:
```

Comment:
```suggestion
The CSS Languages Features extension also provides a formatter. The formatter works with CSS, LESS and SCSS. It is implemented by the [JS Beautify library](https://github.com/beautify-web/js-beautify) and comes with the following settings:
```

Reply:
Good catch. I'll fix and push once you're done in case there is anything else to tune